doordeck-sdk/build.gradle.kts

```diff
@@ -289,4 +288,12 @@ tasks.register<Zip>("zipXCFramework") {
 
 tasks.named("assembleDoordeckSDKReleaseXCFramework").configure {
     finalizedBy("zipXCFramework")
-}
\ No newline at end of file
+}
+
+// Disable source map generation
+tasks.withType<KotlinJsCompile>().configureEach {
+    compilerOptions {
+        sourceMap = false
```

Comment:
Do we not want source maps?

Reply:
To be honest, I was just trying to reduce the size of the Node package. I'm not sure if those files are useful at all, but the overall size has been reduced

Reply:
They aren't loaded by default, so I'm not sure the size reduction is meaningful here - they are typically loaded when used in a browser and you go into the developer console, it will load the source map at that point to show the developer the full version of the code rather than the minified version.